include/boost/geometry/strategies/cartesian/azimuth_cartesian.hpp

```diff
@@ -0,0 +1,64 @@
+// Boost.Geometry (aka GGL, Generic Geometry Library)
+
+// Copyright (c) 2016 Oracle and/or its affiliates.
+// Contributed and/or modified by Vissarion Fisikopoulos, on behalf of Oracle
+
+// Use, modification and distribution is subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_GEOMETRY_STRATEGIES_CARTESIAN_AZIMUTH_CARTESIAN_HPP
+#define BOOST_GEOMETRY_STRATEGIES_CARTESIAN_AZIMUTH_CARTESIAN_HPP
+
+#include <boost/geometry/core/tags.hpp>
+
+namespace boost { namespace geometry
+{
+
+namespace strategy { namespace azimuth
+{
+
+template
+<
+    typename CalculationType
+>
+class azimuth_cartesian
+{
+public :
+
+    inline azimuth_cartesian()
+    {}
+
+    inline void apply(CalculationType const& lon1_rad,
+                      CalculationType const& lat1_rad,
+                      CalculationType const& lon2_rad,
+                      CalculationType const& lat2_rad,
+                      CalculationType& a1,
+                      CalculationType& a2) const
+    {
+        // not implemented
```

Comment:
If `azimuth::services::default_strategy` had static/mpl assert this whole file wouldn't be needed.

As an alternative we could simply implement it. Wouldn't it be something like:

    a1 = a2 = atan2(y2 - y1, x2 -x1);

?